service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/DraftCMOController.java

```diff
@@ -67,9 +68,9 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody Call
 
     @PostMapping("/mid-event")
     public AboutToStartOrSubmitCallbackResponse handleMidEvent(
-        @RequestBody CallbackRequest callbackRequest,
-        @RequestHeader("authorization") String authorization,
-        @RequestHeader("userId") String userId) {
+        @RequestHeader(value = "authorization") String authorization,
```

Comment:
May I ask you to avoid unnecessary changes? They just make code review slower.

Reorder you made brings no value. Exposing attribute name (value) makes it unnecessarily verbose.

Reply:
thanks @geek-soft. Reverted this as the order was initially incorrect and resulted in the attached error
<img width="1575" alt="error" src="https://user-images.githubusercontent.com/6439798/69992078-d1319800-1540-11ea-97a0-247b503928e9.png">